handlers_test.go

```diff
@@ -179,16 +212,20 @@ func TestStatus_assertValidCodes(t *testing.T) {
 	srv := testServer()
 	defer srv.Close()
 
+	methods := []string{http.MethodGet, http.MethodPost, http.MethodPut, http.MethodDelete}
+
 	codes := []int{200, 201, 202, 203, 204, 205, 206, 207, 208, 226,
 		300, 301, 302, 303, 304, 305, 307, 308, 400, 401, 402, 403, 404, 405, 406,
 		407, 408, 409, 410, 411, 412, 413, 414, 415, 416, 417, 418, 422, 423, 424,
 		426, 428, 429, 431, 451, 500, 501, 502, 503, 504, 505, 506, 507, 508, 510, 511}
 
-	for _, code := range codes {
-		u := fmt.Sprintf("%s/status/%d", srv.URL, code)
-		resp, err := noFollowGet(noRedirectClient(), u)
-		require.Nil(t, err, u)
-		require.Equal(t, code, resp.StatusCode, u)
+	for _, method := range methods {
+		for _, code := range codes {
+			u := fmt.Sprintf("%s/status/%d", srv.URL, code)
```

Comment:
if `u` is the message to require.* methods, consider  adding `method` in this message too

I see you do `%s %s, method u` below, just merge them here :)

Reply:
`u` is also used as a url, I'll leave it as is and make sure we include the method on all the `require.*` assertions.